bookfinder/tsconfig.json

```diff
@@ -23,6 +23,6 @@
       "@/*": ["./src/*"]
     }
   },
-  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx", ".next/types/**/*.ts"],
+  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx", ".next/types/**/*.ts", "src/app/components/SearchInput.tsx"],
```

Comment:
Why did you add this line

Reply:
i dont think itr was added intentionally, because i had nothing to do with the tsconfig.json file. probably it was added when i ran "npm run build"

Reply:
I don't think you need to run `npm run build` before pushing your code. You can remove this since `.tsx` is already captured in the flags.